### PR TITLE
fix(manager-pci-common): update peer dependencies (mrc without build)

### PR DESCRIPTION
**Pull request by aboungnaseng-ovhcloud**
ref: MANAGER-14622

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix 14822
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Update peer dependencies in pci common, manager react components, react shell client

### CODE REVIEW

File: packages/manager-react-components/package.json [hunk 1]
```diff
@@ -117,9 +98,12 @@
     "vitest": "^1.2.0",
     "zustand": "^4.5.5"
   },
-  "optionalDependencies": {
+  "peerDependencies": {
+    "@ovh-ux/manager-core-api": "^0.9.0-alpha.0",
+    "@ovh-ux/manager-react-shell-client": "1.0.0",
     "i18next": "^23.8.2",
     "react": "^18.2.0",
+    "react-use": "^17.5.0",
```
Comment (anooparveti) [inline packages/manager-react-components/package.json:106]: **nitpick:** This will cause installation errors as not all the apps are using `react-use`.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/core/shell-client/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ovh-ux/manager-react-shell-client",
-  "version": "0.8.0-alpha.0",
+  "version": "1.0.0",
```
Comment (anooparveti) [inline packages/manager/core/shell-client/package.json:3]: **question:** Whats the need to manually modify the version?

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/core/shell-client/package.json (hunk 2 above)
Comment (aboungnaseng-ovhcloud) [reply packages/manager/core/shell-client/package.json:3]: my pr is not ready yet

Comment (anooparveti) [reply packages/manager/core/shell-client/package.json:3]: You can mark is as Draft then.

File: packages/manager-react-components/package.json (hunk 1 above)
Comment (aboungnaseng-ovhcloud) [reply packages/manager-react-components/package.json:106]: 👍

**Review by chipp972 — APPROVED** [review]

**Review by SimonChaumet — APPROVED** [review]

**Pull request merged**